core/server/master/src/test/java/alluxio/master/block/BlockMasterTest.java

```diff
@@ -76,8 +76,6 @@
   private static final Map<Block.BlockLocation, List<Long>> NO_BLOCKS_ON_LOCATION
       = ImmutableMap.of();
   private static final Map<String, StorageList> NO_LOST_STORAGE = ImmutableMap.of();
-  private static final Block.BlockLocation BLOCK_LOCATION = Block.BlockLocation.newBuilder()
-      .setTier(Constants.MEDIUM_MEM).setMediumType(Constants.MEDIUM_MEM).build();
```

Comment:
The manually simulated `BlockLocation` objects now should contain the correct worker IDs. There can be more tests that I haven't found, that need to be updated.